sdk/export/metric/metric.go

```diff
@@ -16,11 +16,33 @@ package metric // import "go.opentelemetry.io/otel/sdk/export/metric"
 
 import (
 	"context"
+	"sync/atomic"
 
 	"go.opentelemetry.io/otel/api/core"
 	"go.opentelemetry.io/otel/api/metric"
 )
 
+const (
+	// reserved ID for the noop label exporter
+	noopLabelExporterID int64 = 1 + iota
```

Comment:
This is probably my fault, but do we want to call these encoder IDs instead of exporter IDs?

Reply:
Nah, not your fault. I also tend to type label exporter all the time. :) Will fix.

Reply:
Looks like I have made this mistake like everywhere. Bonus points for me for consistency, I guess…

Reply:
See it yourself - https://github.com/open-telemetry/opentelemetry-go/pull/574/commits/4d88d811b807ab23521cab602a7726948e328f62.